azure-toolkit-libs/azure-toolkit-common-lib/src/main/java/com/microsoft/azure/tools/common/exception/CommandExecuteException.java

```diff
@@ -0,0 +1,9 @@
+package com.microsoft.azure.tools.common.exception;
+
+public class CommandExecuteException extends RuntimeException {
+    private static final long serialVersionUID = 4582230448665092548L;
+
+    public CommandExecuteException(String message) {
+        super(message);
+    }
+}
```

Comment:
use RuntimeException directly ?

Reply:
No, the exception class name will indicate the reason, use RuntimeException  will cause confusion